inc/sccm.class.php

```diff
@@ -586,6 +586,8 @@ static function executePush($task) {
                $xmlFile = simplexml_load_file($REP_XML.$tab['MachineID'].'.ocs');
                if ($xmlFile !== false) {
                   $ch = curl_init();
+                  curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
```

Comment:
can you add in the configuration an option allowing the administrator to choose the value